src/assets/wise5/classroomMonitor/dataExport/strategies/LabelComponentDataExportStrategy.ts

```diff
@@ -0,0 +1,78 @@
+import { AbstractComponentDataExportStrategy } from './AbstractComponentDataExportStrategy';
+
+export class LabelComponentDataExportStrategy extends AbstractComponentDataExportStrategy {
+  maxNumLabels: number;
+
+  constructor(nodeId: string, component: any, additionalParams: any) {
```

Comment:
Create new ComponentDataExportParams interface to type additionalParams?
Type ComponentContent?
```suggestion
  constructor(nodeId: string, component: ComponentContent, additionalParams: ComponentDataExportParams) {
```